hopo/studiengebuehren.tex

```diff
@@ -78,25 +157,76 @@ \subsection*{Verwendung der QuaSiMi}
 \iffalse
```

Comment:
Warum hier auch ein Iffalse? In den vorherigen Absätzen steht dass hier ein Exkurs sein soll. Dann sollte man hier mindestens ein Link zu mehr Infos hinzufügen.